File: Lista1/Java/ex_8.java
import java.util.Scanner;
public class ex_8
{
	public static void main(String[] args) {
	    Scanner sc = new Scanner(System.in);
        System.out.println("Digite um número para saber sua tabuada: ");
        int numero = sc.nextInt();
        System.out.println("Tabuada do número " + numero);
	    for (int c = 0; c<=100; c++) {
	      System.out.println(numero + " x " + c + " = " + numero*c);
	    }
	}
}

File: Lista2/ex_8.py
def fatorial(num):
    fatorial = 1
    for i in range(1, num + 1):
        fatorial *= i

    return fatorial

num = int(input('Digite o numero para saber seu fatorial: '))
resultado = fatorial(num)
print(f'O fatorial de {num} é: {resultado}')